assets/src/edit-story/components/form/color/colorPreview.js

```diff
@@ -124,10 +103,12 @@ function ColorPreview({
   const editLabel = __('Edit', 'web-stories');
   const inputLabel = __('Enter', 'web-stories');
 
+  const togglePickerOpen = () => setPickerOpen(() => true);
```

Comment:
Should this be named something else if it's always opening and not a toggle?
Or should it actually close when the button is clicked again? Wondering what's the expected behavior here. As I understand then the focus out handling is closing the calendar when clicking on the button, and right after, the `togglePickerOpen` opens it again, so it would be closing and then opening again right away when clicking the button while the color picker is open already before? Perhaps that's expected though :)

Reply:
Okay, just tested it out.
It looks like there's a slight inconsistency: when clicking the button while the color picker is open, it blinks, however, when choosing a color first and then clicking the button again, it closes the color picker:
![color picker](https://user-images.githubusercontent.com/3294597/78666913-5c001600-78d8-11ea-89d6-7829b407919b.gif)

Also, as can be seen in the gif, the color picker is positioned partly on top of the sidebar currently. Looks like it should be right next to it but not covering, by design.